core/sql/executor/ExExeUtilVolTab.cpp

```diff
@@ -537,15 +539,23 @@ short ExExeUtilCleanupVolatileTablesTcb::work()
 	    char * schemaName = vi->get(0);
 	    retcode =
 	      dropVolatileSchema(masterGlob->getStatement()->getContext(),
-				 schemaName, getHeap(), getGlobals());
+				 schemaName, getHeap(), getGlobals(),
+                                 getDiagsArea());
 	    if (retcode < 0)
 	      {
+                // changes errors to warnings and move on to next schema.
+                if (getDiagsArea())
+                  getDiagsArea()->negateAllErrors();
+
 		// clear diags and move on to next schema.
 		// Remember that an error was returned, we will
 		// return a warning at the end.
 		SQL_EXEC_ClearDiagnostics(NULL);
 		retcode = 0;
 
+                if ((strlen(errorSchemas_) + strlen(schemaName)) < 1000)
+                  strcat(errorSchemas_, schemaName);
```

Comment:
Do you want to stick any delimiter (say a comma or a space) between schema names? Maybe that is already present in schemaName? Also, if they don't all fit, do you want to put some indicator (such as "...") at the end?